orchagent/port.h

```diff
@@ -96,6 +96,16 @@ class Port
     std::vector<sai_object_id_t> m_priority_group_ids;
     sai_port_priority_flow_control_mode_t m_pfc_asym = SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_COMBINED;
     uint8_t m_pfc_bitmask = 0;
+    /*
+     * Following two bit vectors are used to lock
+     * the PG/queue from beeing changed in BufferOrch.
+     * The use case scenario is when PfcWdZeroBufferHandler
+     * sets zero buffer profile it should protect PG/queue
+     * from beeing overriten in BufferOrch.
+     */
+    std::vector<bool> m_queue_lock;
```

Comment:
>m_queue_lock [](start = 22, length = 12)

In theory it is not a lock, it indicates a state.

Reply:
It is not a PFC storm/nostorm state of the queue in case of PfcWdOrch uses acl handler and I think there should be no restriction to change pg or queue during storm when using acl handler?
Could you suggest more suitable stat definition for this case?
In general case it has nothing to do with PFC state of the queue, its just a set of bit flags used to protect the queue.